roles/ovirt-external-providers/README.md

```diff
@@ -21,13 +21,13 @@ The items in `external_providers` list can contain the following parameters:
 | Name                   | Default value       | Description                                                                      |
 |------------------------|---------------------|----------------------------------------------------------------------------------|
 | name                   | UNDEF (Required)    | Name of the external provider.                                                   |
-| state                  | present             | State of the external provider.                                                  |
+| state                  | present             | State of the external provider. Values can be: <ul><li>present</li><li>absent</li></ul>|
 | type                   | UNDEF (Required)    | Type of the external provider. Values can be: <ul><li>os_image</li><li>network</li><li>os_volume</li><li>foreman</li></ul>|
-| url                    | UNDEF (Required)    | URL where external provider is hosted.                                          |
+| url                    | UNDEF (Required)    | URL where external provider is hosted. Not required if state is absent          |
```

Comment:
I think better to remove the `(Requrired)` from default value. and in description say Required if state is present.

Reply:
fixed